docs/privileged-identity-management-requests.md

```diff
@@ -0,0 +1,13 @@
+# Privileged Identity Management (PIM) requests
+
+Accessing production resources requires elevated privileges, we do this through
+Azure’s Privileged Identity Management (PIM) request system.
+
+To make a PIM request:
+
+- Visit
+  https://portal.azure.com/#blade/Microsoft_Azure_PIMCommon/ActivationMenuBlade/azurerbac
+- Activate the 'Contributor' role for the environment you want to access
+- Give a reason for your request and submit
+- For the production environment, you will have to wait until this has been
```

Comment:
I'm going to add something about where in Azure these requests appear / who receives emails.

Reply:
https://portal.azure.com/?Microsoft_Azure_PIMCommon=true#blade/Microsoft_Azure_PIMCommon/ApproveRequestMenuBlade/azurerbac

Reply:
Added